adafruit_rgb_display/ssd1351.py

```diff
@@ -1,5 +1,6 @@
+"""A simple driver for the SSD1351-based displays."""
```

Comment:
Testing with SSD1351 display fails - results in static looking pixellation using example in code.. Tested successfully with current version of this file with same example..

Reply:
@kattni thanks for testing this.  I trusted pylint too much and deleted some important \_\_init\_\_ code.  put it back in the two you have identified and several others.

Reply:
Also notice the internal identifiers were only in the rgb.py file but also should be in all .py files.  So I added those required lines.

Build is not pylinting the examples file correctly.